IMOS/imosParameters.txt

```diff
@@ -53,11 +53,13 @@ DIR_MAG,             0, from_direction,
 DIRECTION,           0, direction_of_the_profile,                                                                 ,              ,              ,                                  ,  ,         ,         ,         char
 DIRT,                0, to_direction,                                                                             degree,        clockwise,     true north,                        E, 999999.0, 0.0,      360.0,    float
 DIST_ALONG_BEAMS,    0, distance_from_sensor_along_beams,                                                         m,             ,              sensor,                            Z, 999999.0, -12000.0, 12000.0,  float
-DOX1,                1, mole_concentration_of_dissolved_molecular_oxygen_in_sea_water,                            umol l-1,      ,              ,                                  O, 999999.0, 0.0,      900000.0, float
-DOX2,                1, moles_of_oxygen_per_unit_mass_in_sea_water,                                               umol kg-1,     ,              ,                                  O, 999999.0, 0.0,      880000.0, float
+DOX,                 0, volume_concentration_of_dissolved_molecular_oxygen_in_sea_water,                          ml l-1,        ,              ,                                  O, 999999.0, 0.0,      200.0,    float
+DOX1,                1, mole_concentration_of_dissolved_molecular_oxygen_in_sea_water,                            umol l-1,      ,              ,                                  O, 999999.0, 0.0,      1000.0, float
+DOX2,                1, moles_of_oxygen_per_unit_mass_in_sea_water,                                               umol kg-1,     ,              ,                                  O, 999999.0, 0.0,      1000.0, float
 DOXS,                1, fractional_saturation_of_oxygen_in_sea_water,                                             percent,       ,              ,                                  O, 999999.0, ,         ,         float
 DOXY,                1, mass_concentration_of_oxygen_in_sea_water,                                                kg m-3,        ,              ,                                  O, 999999.0, 0.0,      29.0,     float
 DOXY_TEMP,           1, temperature_of_sensor_for_oxygen_in_sea_water,                                            degrees_Celsius,,             ,                                  T, 999999.0, 0.0,      50.0,     float
+OXSOL,               0, moles_of_oxygen_per_unit_mass_in_sea_water_at_saturation,                                 umol kg-1,     ,              ,                                  O, 999999.0, 0.0,      1000.0, float

```

Comment:
So this is what teos-10 function calls it,

```
USAGE:

O2sol = gsw_O2sol_SP_pt(SP,pt)

DESCRIPTION:

Calculates the oxygen, O2, concentration expected at equilibrium with  
air at an Absolute Pressure of 101325 Pa (sea pressure of 0 dbar) 
including saturated water vapor.  This function uses the solubility coefficients 
derived from the data of Benson and Krause (1984), as fitted by Garcia 
and Gordon (1992, 1993).

Note that this algorithm has not been approved by IOC and is not work 
from SCOR/IAPSO Working Group 127. It is included in the GSW
Oceanographic Toolbox as it seems to be oceanographic best practice.

INPUT:

SP  =  Practical Salinity  (PSS-78)                         [ unitless ]
pt  =  potential temperature                                   [ deg C ]
       with reference sea pressure (pr) = 0 dbar.

SP & pt need to have the same dimensions.

OUTPUT:

O2sol = solubility of oxygen                                 [ umol/kg ]
```

So could we just call it oxygen_solubility as a long name, with a comment calculated from teos-10 function gsw_O2sol_SP_pt(SP,pt)